tests/unit/store/asserts-test.js

```diff
@@ -0,0 +1,36 @@
+import {createStore} from 'dummy/tests/helpers/store';
+
+import {module, test} from 'qunit';
+
+module("unit/store/asserts - DS.Store methods produce useful assertion messages");
+
+const MODEL_NAME_METHODS = [
+  'createRecord',
+  'findRecord',
+  'findByIds',
+  'peekRecord',
+  'hasRecordForId',
+  'recordForId',
+  'query',
+  'queryRecord',
+  'findAll',
+  'peekAll',
+  'filter',
+  'recordIsLoaded',
+  'modelFor',
+  'modelFactoryFor',
+  'normalize',
+  'adapterFor',
+  'serializerFor'
+];
+
+test("Calling Store methods with no type asserts", function(assert) {
```

Comment:
Since #4223 the tests are also run in the `production` environment, where the assertions are stripped. That's why this test is failing.

You need to use `testInDebug` here instead of `test`:

```js
import testInDebug from 'dummy/tests/helpers/test-in-debug';
```

Sorry you fell over this. This is my fault, I need to update `CONTRIBUTING.md` ASAP.